lib/src/native/rtc_video_view_impl.dart

```diff
@@ -55,7 +58,7 @@ class RTCVideoView extends StatelessWidget {
                 transform: Matrix4.identity()..rotateY(mirror ? -pi : 0.0),
                 alignment: FractionalOffset.center,
                 child: videoRenderer.textureId != null
-                    ? Texture(textureId: videoRenderer.textureId)
+                    ? Texture(textureId: videoRenderer.textureId, filterQuality: filterQuality)
```

Comment:
I propose to split more than one parameter passed to the constructor from one line into several ones. It will be easy to read. So again let's add `,` at the end 😊.